src/site/pages/digital/src/containers/App/index.tsx

```diff
@@ -96,6 +96,7 @@ export const App = ((store: AppStore) => {
 
 
     return function AppView() {
+        warn_compatibility();
```

Comment:
this will check every time the app renders, you should put this in the `index.tsx` at the root of `src` where the digital page starts